data/plugin-nav-data.json

```diff
@@ -13,9 +13,30 @@
     "title": "Which SDK should I use?",
     "path": "which-sdk"
   },
+  {
+    "title": "Publishing to the Registry",
+    "href": "/registry/providers/publishing"
+  },
+  {
+    "title": "Terraform Integration Program",
+    "href": "/docs/partnerships"
+  },
+  {
+    "title": "Community Forum",
+    "href": "https://discuss.hashicorp.com/c/terraform-providers/tf-plugin-sdk/43"
+  },
+
+  {
+    "heading": "Plugin SDKs and Libraries"
+  },
+  { "title": "SDKv2", "href": "sdkv2/sdkv2-intro" },
+  { "title": "Framework", "href": "framework" },
+  { "title": "Logging", "href": "log" },
   {
     "title": "SDKv2",
+    "hidden": true,
```

Comment:
We currently don't have a means to exclude directories, and our nav-data validation imposes a 2-way contract that nav nodes and files must be _linked_ (they must both exist).